File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/remote/response/TvShowResponse.java
package com.ryanrvldo.moviecatalogue.data.source.remote.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.ryanrvldo.moviecatalogue.data.source.model.TvShow;

import java.util.ArrayList;

import lombok.Data;

@Data
public class TvShowResponse {
    @SerializedName("page")
    @Expose
    private int page;

    @SerializedName("results")
    @Expose
    private ArrayList<TvShow> tvShowItems;

    @SerializedName("total_pages")
    @Expose
    private int totalPages;

    @SerializedName("total_results")
    @Expose
    private int totalResults;
}

File: app/src/main/kotlin/com/ryanrvldo/moviecatalogue/data/remote/response/TvShowResponse.kt
package com.ryanrvldo.moviecatalogue.data.remote.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.ryanrvldo.moviecatalogue.data.model.TvShow

data class TvShowResponse(
    @SerializedName("page") @Expose val page: Int,
    @SerializedName("results") @Expose val tvShowItems: List<TvShow>,
    @SerializedName("total_pages") @Expose val totalPages: Int,
    @SerializedName("total_results") @Expose val totalResults: Int
)